### PR TITLE
Terms and Conditions in Sign-up Flow

### PR DESCRIPTION
**Pull request by emilyzhong**
## Terms and Conditions agreement
[Asana Ticket](https://app.asana.com/0/860297512787618/1110475747308486/f)
* Added checkbox / field for artists and buyers in sign-up flow to accept terms and conditions
  * [Referenced used because I will probably forget](https://gist.github.com/withoutwax/46a05861aa4750384df971b641170407) 
* Validation for terms and conditions checkbox

TODO: Add actual terms and conditions content

### Related PRs
n/a

### Migrations
* added column `terms_and_conditions` to `Artist`
* added column `terms_and_conditions` to `Buyer`

### Tests Performed, Edge Cases
n/a

### Screenshots
Sign up form: 
image

Example of validation error:
screen shot 2019-02-26 at 12 24 18 am

CC: @by-co

### CODE REVIEW

File: app/assets/stylesheets/pages/Login.scss [hunk 1]
```diff
@@ -32,4 +36,21 @@
       background-color: $mustard;
     }
   }
-}
\ No newline at end of file
+
+  .terms {
```
Comment (by-co) [inline app/assets/stylesheets/pages/Login.scss]: Hmm this looks like exclusively Tachyons, why not have it inline I'm curious? 😮

File: app/assets/javascripts/signin.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+nextPage = () => {
+	$(".signup.part1").hide();
```
Comment (by-co) [inline app/assets/javascripts/signin.js]: Wow magical 👏

File: app/assets/stylesheets/pages/Login.scss [hunk 3]
```diff
@@ -23,9 +23,14 @@
 			text-decoration: underline;
 		}
   }
+
   input {
     border-style: none;
   }
+
+  input.invalid {
+    background-color: $red;
```
Comment (by-co) [inline app/assets/stylesheets/pages/Login.scss]: The red background might be a little aggressive, I'll make some error state mocks for you + help text

File: app/views/artists/registrations/new.html.erb [hunk 4]
```diff
@@ -1,38 +1,46 @@
...
         <%= f.check_box :terms_and_conditions %>
         <%= f.label :terms_and_conditions, "I agree to SFAI's Terms and Conditions" %>
       </div>
-
-      <div class="flex justify-between">
-        <%= render "artists/shared/links" %>
-        <%= f.submit "Sign up", class: "button-primary bg-mustard w4" %>
-      </div>
-    <% end %>
+      <%= f.submit "Sign up", class: "button-primary bg-mustard w4" %>
```
Comment (by-co) [inline app/views/artists/registrations/new.html.erb]: Can we do inline validation for the checkbox? Like if the user doesn't check the box, don't let them submit the form?

File: app/assets/stylesheets/pages/Login.scss (hunk 3 above)
Comment (by-co) [reply app/assets/stylesheets/pages/Login.scss]: We'll save this for another PR haha
